socialhome/streams/app/components/StreamElement.vue

```diff
@@ -6,126 +6,66 @@
             v-bind="author"
             :is-user-author="isUserAuthor"
             :is-user-local="isUserLocal"
-            :current-browsing-profile-id="currentBrowsingProfileId"
-            :is-user-authenticated="isUserAuthenticated"
         >
         </author-bar>
-        <div class="grid-item-bar d-flex justify-content-start">
+        <reactions-bar
+            :id="id"
+            :replies-count="repliesCount"
+            :shares-count="sharesCount"
+            :has-shared="hasShared"
+        >
             <div class="mt-1">
-                <a :ref="contentUrl" :title="timestamp">
+                <a :href="contentUrl" :title="timestamp" variant="link" class="unstyled-link">
```

Comment:
It's an `a` tag, does it need `variant`?

Reply:
No, it does not. It's... It's... It's a... typo...